ansible/roles/request-ocs/files/ocs_install.sh

```diff
@@ -152,26 +175,52 @@ spec:
     resources: {}
 EOF
 
-# check if storage classes have been created
-echo "checking for Storage Classes"
-scCounter=0
-rc=1
-until [ $rc -eq 0 ]
-do
-  echo "Waiting for storage classes to come up: $(date)"
-  sleep 10
-  ((scCounter ++))
-  if [ $scCounter -eq 60 ]; then
-   exit 1
-  fi
-  oc get sc --no-headers | cut -f1 -d' ' | grep noobaa >/dev/null
-  rc=$?
-done
-oc get sc
-echo "setDefault = $setDefault"
-if [ "$setDefault" == "True" ]; then
-   defStgClass=$(oc get sc -o json | jq -r '.items[].metadata | select(.annotations["storageclass.kubernetes.io/is-default-class"] == "true") | .name')
-   oc patch storageclass $defStgClass -p '{"metadata": {"annotations":{"storageclass.kubernetes.io/is-default-class":"false"}}}'
-   oc patch storageclass ocs-storagecluster-cephfs -p '{"metadata": {"annotations":{"storageclass.kubernetes.io/is-default-class":"true"}}}'
-   oc get sc
-fi
+# # check if storage classes have been created
+# echo "checking for Storage Classes"
+# scCounter=0
+# rc=1
+# until [ $rc -eq 0 ]
+# do
+#   echo "Waiting for storage classes to come up: $(date)"
+#   sleep 10
+#   ((scCounter ++))
+#   if [ $scCounter -eq 60 ]; then
+#    exit 1
+#   fi
+#   oc get sc --no-headers | cut -f1 -d' ' | grep noobaa >/dev/null
+#   rc=$?
+# done
+# oc get sc
```

Comment:
Need to keep the code that set the default storageclass based on setDefault=True lines 193 to 200,

Your code should only replace 178 to 192

Reply:
I'm not sure what is exactly wrong ... have not had the time to investigate... but have not been able to get these changes to work on our VMware setup.

Reply:
@LeslieKuo have you had time to look over walts comments?